src/components/Calendars/NutritionistCalendar.tsx

```diff
@@ -0,0 +1,128 @@
+'use client';
+import { Menu, Transition } from '@headlessui/react';
+import { add, eachDayOfInterval, endOfMonth, format, getDay, isEqual, isSameDay, isSameMonth, isToday, parse, parseISO, startOfToday } from 'date-fns';
+import React, { Fragment, useState } from 'react';
+import { ChevronLeftIcon, ChevronRightIcon, DotsVerticalIcon } from '@radix-ui/react-icons';
+import { pt } from 'date-fns/locale';
+import Image from 'next/image';
+import { Appointment } from '@/types/Appointment';
+import Link from 'next/link';
+
+let colStartClasses = ['', 'col-start-2', 'col-start-3', 'col-start-4', 'col-start-5', 'col-start-6', 'col-start-7'];
+
+export type NutritionistCalendarProps = {
+    appointments: Appointment[];
+};
+
+export default function NutritionistCalendar({ appointments }: NutritionistCalendarProps) {
+    let today = startOfToday();
+    let [selectedDay, setSelectedDay] = useState(today);
+    let [currentMonth, setCurrentMonth] = useState(format(today, 'MMM-yyyy'));
+    let firstDayCurrentMonth = parse(currentMonth, 'MMM-yyyy', new Date());
+
+    let days = eachDayOfInterval({
+        start: firstDayCurrentMonth,
+        end: endOfMonth(firstDayCurrentMonth),
+    });
+
+    function previousMonth() {
+        let firstDayNextMonth = add(firstDayCurrentMonth, { months: -1 });
+        setCurrentMonth(format(firstDayNextMonth, 'MMM-yyyy'));
+    }
+
+    function nextMonth() {
+        let firstDayNextMonth = add(firstDayCurrentMonth, { months: 1 });
+        setCurrentMonth(format(firstDayNextMonth, 'MMM-yyyy'));
+    }
+
+    let selectedDayMeetings = appointments.filter((consultation) => isSameDay(parseISO(consultation.attributes.date), selectedDay));
```

Comment:
Why using let here?

Reply:
you are right, I was putting let on everything, changed to const now